briefcase-guide.rst

```diff
@@ -0,0 +1,12 @@
+******************************
+ODK Briefcase
+******************************
+
+:dfn:`ODK Briefcase` is a Java application for fetching and pushing forms and their contents.It can be used to gather and aggregate data from ODK Collect when there is no internet connectivity or you don't want to set up an ODK Aggregate Server.
+
+:doc:`Typical Usage <briefcase-forms>`
```

Comment:
I don't think this should be a link. Instead, the actions in the list below should link to the sections that describe each feature.